nautobot/users/tests/test_api.py

```diff
@@ -39,9 +39,9 @@ class UserTest(APIViewTestCases.APIViewTestCase):
         },
         {
             "username": "User_6",
-            "password": "password6",
         },
     ]
+    update_data = {"username": "User_22"}
```

Comment:
Do we need to add the rest of the user fields (`is_superuser`, `last_login`, `password`, etc.) to the `create_data`/`update_data` properties on this test?

Reply:
I don't think it's critical, since with the new round-trip csv test case we're exercising every field that declares itself as writable for the `create` case, but it would be good to add just to assert what fields we *expect* to be writable. I'll try to remember to tackle this on Monday - it does fall into a broader category of expanded API test data that's probably needed/desirable after #3500 expanded so many of the serializers' fields.